app/models/job_template.rb

```diff
@@ -72,8 +73,10 @@ def self.import(contents, options = {})
 
       template = existing || self.new
       template.sync_inputs(metadata.delete('template_inputs'))
-      template.assign_attributes(metadata.merge(:template => contents.gsub(/<%\#.+?.-?%>\n?/m, '')).merge(options))
+      template.sync_foreign_input_sets(metadata.delete('foreign_input_sets'))
+      template.assign_attributes(metadata.merge(:template => contents.gsub(/<%\#.+?.-?%>\n?/m, '')).merge(options).except('feature'))
       template.assign_taxonomies if template.new_record?
+      template.sync_feature(metadata.delete('feature'))
```

Comment:
Hm, I see - I'd think this could be done differently so the feature wouldn't need persisting. It'll cascade save and save the job template, but I don't want that done in `import`. (There's a reason for it when I want to write the community-templates plugin, I need import *not* to save, so it can do some diff stuff first.)

Anywho, I filed http://projects.theforeman.org/issues/13746 for that.